packages/core/src/components/Tabs/index.tsx

```diff
@@ -171,10 +183,19 @@ export default withBoundary('Tabs')(
       display: 'flex',
     },
 
-    nav_borderless: {
+    nav_noBorder: {
       borderColor: color.clear,
     },
 
+    nav_secondary: {
+      padding: unit / 2,
+      alignItems: 'center',
+      overflow: 'auto',
+      borderWidth: 0,
+      borderRadius: ui.borderRadius,
+      backgroundColor: color.accent.bgHover,
```

Comment:
This gray BG should be an app concern, not the components.

Reply:
@milesj that's a fair point... but then should the padding be made configurable? The padding does not make much sense without the bgColor.

Reply:
I would agree. I can foresee the gray bg/padding causing issues for other integration locations. Probably best to leave it up to the consumer.